simple-repository-API/Controllers/RepositoryController.cs

```diff
@@ -17,5 +17,11 @@ public Student GetStudent(int id)
         {          
             return _studentRepository.GetStudent(id);
         }
+
+        [HttpPost("StudentsList")]
+        public List <Student> GetStudentsLists()
```

Comment:
szczegół ale usuńmy spacje w List <

Reply:
Samo GetStudents. To Lists jest nie potrzebne + nazwa Lists wskazuje jakby to były listy a nie lista